exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/NestedLoopJoinPrule.java

```diff
@@ -84,8 +88,14 @@ public void onMatch(RelOptRuleCall call) {
     if (!settings.isNestedLoopJoinEnabled()) {
       return;
     }
-
-    final DrillJoinRel join = (DrillJoinRel) call.rel(0);
+    int[] joinFields = new int[2];
+    DrillJoinRel join = (DrillJoinRel) call.rel(0);
+    // If right outer join on simply equi join convert it to left outer join. We only support left outer NLJ as of now
+    if (join.getJoinType() == JoinRelType.RIGHT
```

Comment:
Can you do this change as part of a separate JIRA since it is unrelated to statistics.

Reply:
Does DRILL-5214 need to be fixed in order to ship stats functionality ?

Reply:
This change is what we wanted to do as part of DRILL-5214 since it is unrelated to statistics. I will just close that JIRA as fixed once this goes in.